lib/rails/graphql/field.rb

```diff
@@ -226,6 +226,21 @@ def inspect(extra = '') # :nodoc:
         "#<GraphQL::Field @owner=\"#{owner.name}\" #{name}#{args}:#{extra}#{dirs}>"
       end
 
+      # Update the null value if possible
+      def set_null(value)
```

Comment:
I think it's better to have a function named `required!` or something like that, so you don't have to do any checking, just st `@null = false`.